pkgs/top-level/all-packages.nix

```diff
@@ -21229,11 +21229,10 @@ in
   };
 
   inherit (callPackage ../applications/virtualization/docker {})
-    docker_18_09
-    docker_19_03;
+    docker_20_10;
 
-  docker = docker_19_03;
-  docker-edge = docker_19_03;
+  docker = docker_20_10;
+  docker-edge = docker_20_10;
```

Comment:
Maybe it would be better to move this to `aliases.nix` file :thinking: ?

Reply:
This could change in the future when they start builds for 21.04.

Reply:
At least to me it seems they stopped with edge releases (can't find a reference for Docker for Linux, but it seems that what happened for Windows/macOS at least).